src/utils/endaoment.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, Optional
+from urllib.parse import urlencode
+
+import requests
+
+
+def search_nonprofit_orgs(
```

Comment:
This is not located appropriately. The view is in `nonprofit` but the core functionality resides in the generic `utils` package. Ideally this is co-located with the rest of the code.